docs/en/reference/custom-configuration.rst

```diff
@@ -68,3 +68,62 @@ Once you have your custom integration setup, you can modify it to look like the
     $cli->run();
 
 :ref:`Next Chapter: Migrations Events <events>`
+
+
+It is possible to use multiple entity managers or connections, this is a way to configure your application:
+
+
+.. code-block:: php
+
+    #!/usr/bin/env php
+    <?php
+
+    require_once __DIR__.'/vendor/autoload.php';
+
+    use Doctrine\DBAL\DriverManager;
+    use Doctrine\Migrations\Configuration\Configuration;
+    use Doctrine\Migrations\Configuration\Configuration\ExistingConfiguration;
+    use Doctrine\Migrations\Configuration\Connection\ConnectionRegistryConnection;
+    use Doctrine\Migrations\DependencyFactory;
+    use Doctrine\Migrations\Tools\Console\Command;
+    use Symfony\Component\Console\Application;
+
+    $connection1 = DriverManager::getConnection([...]);
+    $connection2 = DriverManager::getConnection([...]);
+
+    $connectionRegistry = new class (
```

Comment:
What does `class` represent here?

Reply:
good catch, i forgot the "extends" part, now I have added it